### ISSUE TITLE
Search lat/lon  separated by commas

### ISSUE THREAD

**Issue by anastasiia936** [osmandapp/OsmAnd#14082]
### Description
If the separator between the coordinates is a comma, OsmAnd will not be able to find the desired address.

### How to reproduce?
Check this coordinates: ```38,7905625 -9,793281```
```38,7905625, -9,793281```
<table style="width:100%">
  <tr>
    <th><img src="https://user-images.githubusercontent.com/80623554/163599081-4577f95c-bb81-4001-a4c2-00d487c0b00a.jpg"  width="800" height="auto"/> </th>
 <th><img src="https://user-images.githubusercontent.com/80623554/163599187-b8ee8466-ace1-4c41-ab25-12144711bfb4.png"  width="300" height="auto"/> </th>
  </tr>
</table> 

### Your Environment
OsmAnd Version: 4.1.11
Android/iOS version: Android 9
Device model: Sone Experience XZ1

**Maps used (online or offline):**   offline, updated from 1.04.2022
If you have an issue related to offline maps, tell us the exact name of the map file where the issue occurs and its edition date.

**Comment by basings** [comment]
A comma is usually used as a separator for coordinates, i.e.  38.7905625, -9.793281 not as a decimal point. Is that a language thing?

**Comment by tradavyvy** [comment]
Yes it is, in europe we use a decimal comma instead of a decimal point.
See also #13653

**Comment by basings** [comment]
@tradavyvy where in europe? 

I can't replicate the google maps example from your link. Google uses decimal points and doesn't understand commas either.

**Comment by tradavyvy** [comment]
Google is inconsistent in the use of 'decimal deviders' but if your language is Dutch then the most easy copy-able cordinates are with comma's AND they are separated by comma's.
(I know ... It is weird, but it is Google)
See example:
![Screenshot_20220415-225502_Maps](https://user-images.githubusercontent.com/71978409/163631400-4507cc5d-f23b-4e0d-b2f1-80ecc00092b8.jpg)

**Comment by tradavyvy** [comment]
In the top of the pagebox google uses decimal points and avcomma separator.
But I can not coppy those coordinates.
Only when I 'pull up' the bottom screen I get the coppy-able 'triple comma numbers'
![Screenshot_20220415-230046_Maps](https://user-images.githubusercontent.com/71978409/163631794-0076efd7-1f3d-4586-acee-4044e190d29d.jpg)

**Comment by tradavyvy** [comment]
The problem definitely lies with google. But if osmand would be accepting both comma's and points life would be easier.
🙃

**Comment by basings** [comment]
That sounds like google doesn't want you to copy coordinates

**Comment by tradavyvy** [comment]
That is probably right, google wants everyone to stick to google. But we want us and others to use osmand. But this all diverges from the original request to be able to use coordinates with decimal points AND with decimal commas. And irrespectable wether they are separeted by commas, points, tabs, spaces or horisontal bars ( - ).

**Comment by Lee-Carre** [comment]
@basings

> > in europe we use a decimal comma instead of a decimal point.
> 
> where in europe?

[Wikipedia § Decimal Separator § Usage Worldwide](//www.wikipedia.org/wiki/Decimal_separator#Usage_worldwide)

Apparently most of Europe. It seems popular outside of Europe, too.
Some nations seem to use both / either.
There are other separators, too; Arabic nations have their own (considering that we're using Arabic numerals).
I recall, from browsing through Unicode characters, that some scripts have their own dedicated character for it.

While Google does seem buggy, it doesn't seem that using commas is strictly incorrect.
I'm reminded of [Postel's Law](//www.wikipedia.org/wiki/Postel's_Law), about being liberal in what you accept, for robustness.

**Comment by basings** [comment]
@Lee-Carre thx! The link you shared leads to the general usage of the decimal point/comma not the special use case for coordinates. On the german wikipedia page https://de.m.wikipedia.org/wiki/Geographische_Koordinaten they use a point, the same on the italian page. I couldn't find anything useful in the french, spanish or danish. There must be an international convention. I can't find it.

If, there is no international consensus about how to write coordinates, then it doesn't matter much. If the point notation is internationally accepted and there's no drift by some to deviate, then deviating from the norm isn't that good. It would be bad if people would start using the comma notation just because google doesn't want people to copy paste coordinates. Accept the search query, but correct it automatically or suggest the proper notation - if and only if there is an international consensus about point notation being the uniform notation.

**Comment by tradavyvy** [comment]
The page you quote uses the international (read english) way ofnotation for wsg coordinates.
But the coordinates are simply 2 numbers (degrees north amd east) in decimal notation, and in (most of) europe we use the komma as the decimal separator. 
(So programmers please make this notation acceptable for input use)

**Comment by dmpr0** [comment]
Fixed
